Don't access the net to load public links for the nova and swift projects.
These don't seem to be referenced yet in any case.

--- glance-2011.3.orig/doc/source/conf.py	2011-09-21 21:46:35.000000000 +0100
+++ glance-2011.3/doc/source/conf.py	2011-11-23 15:57:39.113642425 +0000
@@ -45,7 +45,6 @@
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.coverage',
               'sphinx.ext.ifconfig',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.pngmath',
               'sphinx.ext.graphviz',
               'sphinx.ext.todo']
